app/components/m-header.js

```diff
@@ -81,7 +81,7 @@ export default class MHeader extends Component {
   @task
   *loadRoles() {
     this.roles = yield this.store.queryAll('role', {
-      'filter[concept-scheme][:uri:]': CONSTANTS.CONCEPT_SCHEMES.USER_ROLES,
```

Comment:
is this right? we are doing `filter[concept-scheme][:uri:]': CONSTANTS.CONCEPT_SCHEMES.USER_ROLES,` in a few other places aswell.